File: app/src/main/java/org/ngbp/jsonrpc4jtestharness/rsp/keys/model/NotifyParams.java
package org.ngbp.jsonrpc4jtestharness.rsp.keys.model;

import java.util.List;

public class NotifyParams {
    public String msgType;
    public List<Timeout> timeout;
    public class Timeout{
        public String key;
        public Integer time;
    }
}

File: app/src/main/java/org/ngbp/jsonrpc4jtestharness/rpc/drm/model/NotifyParams.kt
package org.ngbp.jsonrpc4jtestharness.rpc.drm.model

import org.ngbp.jsonrpc4jtestharness.rpc.RpcResponse

data class NotifyParams(
        var msgType: String? = null,
        var systemId: String? = null,
        var message: MutableList<Any?>? = null
) : RpcResponse()